app/src/main/java/by/paranoidandroid/notesapp/adapters/NotesAdapter.java

```diff
@@ -11,18 +11,20 @@
 
 import androidx.recyclerview.widget.RecyclerView;
 import by.paranoidandroid.notesapp.R;
-import by.paranoidandroid.notesapp.database.AppDatabase;
 import by.paranoidandroid.notesapp.database.entities.Note;
+import by.paranoidandroid.notesapp.utils.RemoveItemListener;
 
 public class NotesAdapter extends RecyclerView.Adapter<NotesAdapter.ViewHolder> {
     private List<Note> notes;
     private LayoutInflater inflater;
     private Context context;
+    private RemoveItemListener removeItemListener;
 
-    public NotesAdapter(Context ctx) {
+    public NotesAdapter(Context ctx, RemoveItemListener listener) {
```

Comment:
Think, how can do without the listener

Reply:
Unfortunately, I seem to have no idea how to do this in other way... Could you please give me a tip?